Makefile

```diff
@@ -34,7 +34,7 @@ typecheck-pyright:
 
 .PHONY: typecheck-mypy
 typecheck-mypy:
-	uv run mypy --strict tests/typed_agent.py
+	uv run mypy
```

Comment:
Does this cover the `tests/` files? If so, was there a reason we didn't do this sooner?